volume-group-operator/controllers/utils/volumegroupcontent.go

```diff
@@ -64,6 +59,15 @@ func (r *ControllerUtils) generateVolumeGroupContentStatus(groupCreationTime *me
 	}
 }
 
+func (r *ControllerUtils) UpdateVolumeGroupStatus(logger logr.Logger, vgc *volumegroupv1.VolumeGroupContent, groupCreationTime *metav1.Time, ready *bool) error {
+	vgc.Status = r.generateVolumeGroupContentStatus(groupCreationTime, ready)
```

Comment:
I think we should pass the `pvList`, I think with this code it will overwrite the `pvList` and leave it empty

Reply:
done